rib/rib_test.go

```diff
@@ -3346,3 +3346,225 @@ func mustSharedNHRIB(nhNI string) *RIB {
 
 	return r
 }
+
+func TestFlush(t *testing.T) {
+	tests := []struct {
+		desc             string
+		inRIB            *RIBHolder
+		wantErrSubstring string
+	}{{
+		desc: "nh only",
+		inRIB: func() *RIBHolder {
+			r := NewRIBHolder("DEFAULT")
+
+			_, _, err := r.AddNextHop(&aftpb.Afts_NextHopKey{
+				Index: 1,
+				NextHop: &aftpb.Afts_NextHop{
+					ProgrammedIndex: &wpb.UintValue{Value: 42},
+				},
+			}, false)
+			if err != nil {
+				panic(fmt.Sprintf("cannot add next-hop, %v", err))
```

Comment:
maybe t.Fatal is better?

Reply:
Agree -- on all these cases I moved to `t.Fatalf`, including in other cases other than where we have these `mustXXX` patterns that do not have a `testing.T` argument to call `t.Fatalf` on.